### ISSUE TITLE
[TF2] The Face Plante doesn't remove the default hat

### ISSUE THREAD

**Issue by mock--** [ValveSoftware/Source-1-Games#4385]
The images of the Face Plante [in the workshop](https://steamcommunity.com/sharedfiles/filedetails/?id=99948250) show it without the Scout's default hat, but[ as currently implemented,](https://wiki.teamfortress.com/wiki/Face_Plante) the default hat remains and clips into the mask.

**Comment by Chickenman456** [comment]
IIRC this hat is actually implemented incorrectly.... the material looks bright, almost white in-game for some reason

**Comment by mock--** [comment]
Yes. Painting it white actually makes it darker than the unpainted version.

**Comment by AestheticSnow** [comment]
Additionally to not removing the Scout's default hat, the Face Plante hides his dogtags, which based on the Workshop images is not intended. Swap bodygroup removal of dogtags with his default hat.
![250px-Face_Plante](https://github.com/ValveSoftware/Source-1-Games/assets/115660567/a237614e-6294-4354-b517-2ef1b0c02bb8)
![steamworkshop_webupload_previewfile_99948250_preview](https://github.com/ValveSoftware/Source-1-Games/assets/115660567/9b476f89-8029-4509-b934-20bb303f769e)

**Comment by AestheticSnow** [comment]
And something is wrong with its texture. Currently it's much brighter than it looks like on the Workshop images.

**Comment by KimmyTF2** [comment]
As of [November 14, 2024 Patch](https://wiki.teamfortress.com/wiki/November_14,_2024_Patch) it no longer hides the Scout's dog tags, but still leaves his default hat visible.
